src/04-spin-boson-model.md

```diff
@@ -0,0 +1,776 @@
+
+$$H =  \overbrace{A \sigma_z}^{TSS}+ \overbrace{\hbar\omega\left(a^{\dagger}a +\frac{1}{2}\right)}^{bosons} + \overbrace{\frac{\delta}{2}\left( a^{\dagger} + a \right)\sigma_x}^{interaction}$$
+
+The labels A and $\delta$ were chosen because of the path of discovery that we took to get here. From now on, it will be convenient for us to re-write the Hamiltonian with different labels as:
+
+$$H =  \frac{\Delta E}{2} \sigma_z + \hbar\omega\left(a^{\dagger}a +\frac{1}{2}\right) + U\left( a^{\dagger} + a \right)\sigma_x$$
+
+where we recognise $\Delta E$ as the transition energy of the TSS, $\hbar\omega$ the energy of a single boson and $U$ as the strength of the interaction of the TSS with the boson field.
+
+
+In the last tutorial we saw how the physics of spontaneous emission arose from an indirect coupling between the |+> and |-> states of the TSS which was mediated by the interaction with the boson field. This interaction manifested as a Rabi type oscillation between two states of the combined spin boson system (described by |n,±>), namely |0,+> and |1,->.
+
+In general, the result of such interactions are far more complicated than the Rabi type oscillations we are becoming familiar with. We glimpsed this complexity last time when we observed that many bosons modes, with slightly different frequencies, interact with the TSS to produce an irregular pattern of state occupation probabilities. The complexity arises because the combined TSS and field system has many states that interact with each other strongly if their energies are similar.
+
+We can however still apply the ideas we've learnt about TSS's in some special and interesting situations. That is what we'll explore in this notebook.
+
+
+## 4.2 - Stationary states
+
+
+We begin in the same way that we started this tutorial series ([Tutorial 1](https://nbviewer.jupyter.org/github/project-ida/two-state-quantum-systems/blob/master/01-an-isolated-two-state-system.ipynb#1.1---Stationary-state)) by looking for the stationary states of the system. When the system is in one of these states it will remain there for all time. Such states are described by a single constant energy.
+
+To find the states of constant energy, we'll follow what we did in [Tutorial 2](https://nbviewer.jupyter.org/github/project-ida/two-state-quantum-systems/blob/master/02-perturbing-a-two-state-system.ipynb#2.1-Static-perturbation). Specifically, we will calculate the eigenvalues of the Hamiltonian (i.e the energies) and see how they depend on the TSS transition energy $\Delta E$.
+
+What do we expect to find? Recall that in Tutorial 2 (see Fig 3) we discovered an [avoided crossing](https://en.wikipedia.org/wiki/Avoided_crossing) (aka anti-crossing) when the transition energy of the TSS was zero  - this was due to the coupling between the states splitting the energy levels apart.
```

Comment:
I would just overemphasize the difference between Tutorial 2 and 4 saying something like: "Remember that in Tutorial 2 there was no field and the coupling was a direct interaction between states, while here the effective coupling is only an indirect coupling,  mediated by the field"